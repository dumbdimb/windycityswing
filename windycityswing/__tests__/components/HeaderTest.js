import React from 'react';
import renderer from 'react-test-renderer';

import Header from '..\\..\\..\\src\\scripts\\components\\Header';

it('renders correctly', () => {
    expect(renderer.create(<Header/>).toJSON()).toMatchSnapshot();
});