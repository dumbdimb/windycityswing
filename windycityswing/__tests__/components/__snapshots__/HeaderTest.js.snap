// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  href="/WindyCitySwing"
  id="header"
>
  <span
    id="title"
  >
    Windy City Swing
  </span>
  <div
    id="subtitle"
  >
    THE SWING DANCING INFORMATION IN CHICAGO AND CHICAGOLAND AREA
  </div>
</a>
`;
